typings/objection/index.d.ts

```diff
@@ -302,15 +299,14 @@ declare module "objection" {
   export interface QueryBuilder<T> extends QueryBuilderBase<T>, Promise<T[]> { }
 
   interface Insert<T> {
-    <M extends Model[] | Object[]>(modelsOrObjects?: M): QueryBuilder<T>;
-    <M extends ModelOrObject>(modelOrObject?: M): QueryBuilderSingle<T>;
+    (modelsOrObjects?: Array<Partial<T>>): QueryBuilder<T>;
```

Comment:
Also, could you add an example to `tests/ts/examples.ts`? Something as simple as
```
// Verify that insert accepts Partial<Person>:

Person.query()
  .insert({ firstName: "test" })
  .then((p: Person) => console.log("inserted", p))
```

would be great. Thanks.